modules/plottinggl/src/plotters/scatterplotgl.cpp

```diff
@@ -492,24 +492,26 @@ void ScatterPlotGL::objectPicked(PickingEvent *p) {
         }
     };
 
-    if (auto me = p->getEventAs<MouseEvent>()) {
-        if (me->button() == MouseButton::Left) {
-            if (me->state() == MouseState::Release) {
-                // print information on current element
-                logRowData();
+    if (p->getState() == PickingState::Updated) {
+        if (auto me = p->getEventAs<MouseEvent>()) {
```

Comment:
Separate handling of mouse and touch can be remove if we use this?
```
p->getState() == PickingState::Updated && p->getPressState() == PickingPressState::Release &&
        p->getPressItem() == PickingPressItem::Primary
```

Reply:
might be possible